kuma/static/js/wiki-edit.js

```diff
@@ -458,90 +458,120 @@
     // Initialize logic for save and save-and-edit buttons.
     //
     function initSaveAndEditButtons () {
+        var $form = $('#wiki-page-edit');
+        // Handle edits on the translate page
+        if ($form.length == 0 && $('#translate-document').length == 1) {
+            $form = $('#translate-document');
+        }
+
         // Save button submits to top-level
-        $('.btn-save').on('click', function () {
-            if (draftingEnabled) {
+        $form.on('submit', function () {
+            if (supportsLocalStorage) {
                 // Clear any preserved content.
-                clearDraft('publish');
+                clearDraft();
                 clearTimeout(DRAFT_TIMEOUT_ID);
             }
-            $form.attr('action', '').removeAttr('target');
             return true;
         });
 
-        // Save-and-edit submits to a hidden iframe, show loading message in notifier
-        var notifications = [];
-        $('.btn-save-and-edit').on('click', function () {
+        // save and edit attempts ajax submit
+        $('.btn-save-and-edit').on('click', function(event) {
+            // disable form
+            $('#wiki-page-edit').attr('disabled', true);
 
-            notifications.push(mdn.Notifier.growl('Saving changes…', { duration: 0 }));
+            // give user feedback
+            var saveNotification = mdn.Notifier.growl('Saving changes…', { duration: 0 });
 
+            // record event
             mdn.analytics.trackEvent({
-                category: 'Wiki',
-                action: 'Button',
-                label: 'Save and Keep Editing' // now "Publish and keep editing" but keeping label for analytics continuity
-            });
+                    category: 'Wiki',
+                    action: 'Button',
+                    label: 'Save and Keep Editing'
```

Comment:
Please preserve the comment `// now "Publish and keep editing" but keeping label for analytics continuity`